lib/hashie/extensions/strict_key_access.rb

```diff
@@ -24,20 +24,24 @@ def initialize(msg = 'Setting or using a default with Hashie::Extensions::Strict
         end
       end
 
-      # NOTE: This extension would break the default behavior of Hash initialization:
+      # NOTE: Defaults don't make any sense with a StrictKeyAccess.
+      # NOTE: This extension breaks the "default" behavior of Hash initialization,
+      # whereby a default can be set to be used as the value returned when key lookup fails.
       #
-      #     >> a = StrictKeyAccessHash.new(a: :b)
+      # Normal:
+      #
+      #     >> a = Hash.new(123)
       #     => {}
-      #     >> a[:a]
-      #       KeyError: key not found: :a
+      #     >> a["noes"]
+      #     => 123
+      #
+      # Default is so not relevant, it blows up!
```

Comment:
Just to be pedantic this doesn't mean anything and it doesn't "blow up", it raises a KeyError ;) I would say something like "the extension causes the Hash to raise a KeyError even with a default set".